pom.xml

```diff
@@ -67,6 +67,21 @@
     -->
   </dependencies>
   
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <version>3.0.1</version>
+        <configuration>
+          <xmlOutput>true</xmlOutput>
+          <findbugsXmlWithMessages>true</findbugsXmlWithMessages>

```

Comment:
findbugsXmlWithMessages is not used in findbugs-3.x. It may make sense to explicitly enable FindBugs on the verify stage. I'll send a template snippet to the mailing list